src/modules/plus/ebur128/ebur128.c

```diff
@@ -398,12 +398,11 @@ void ebur128_get_version(int* major, int* minor, int* patch) {
   *patch = EBUR128_VERSION_PATCH;
 }
 
-#define VALIDATE_MAX_CHANNELS (64)
```

Comment:
I do not want to change this file because I want to keep it synchronized with the upstream project
https://github.com/jiixyj/libebur128

Reply:
Is the limit in libebur128 defined in a publicly accessible header? If so then we should use that rather than defining our own.

Reply:
Just checked; it's in ebur128.c so the answer is no.

Reply:
Oh wait the whole of it is just incorporated as-is. Nevermind then. It seems it will successfully fail if the number of channels is greater than the limit.